qp_klp/tests/test_klp.py

```diff
@@ -617,29 +619,30 @@ def test_map_sample_names_to_tube_ids(self):
         copy(join(output_dir, 'good-prep-file.txt'),
              join(output_dir, 'Sample_Project.tsv'))
 
-        results = map_sample_names_to_tube_ids(sn_tid_map_by_proj, output_dir)
+        files_list = ['qp_klp/tests/good-prep-file.txt',
+                      'qp_klp/tests/Sample_Project.tsv']
 
-        for project in results:
-            for prep_file in results[project]:
-                df = results[project][prep_file]
+        map_sample_names_to_tube_ids(files_list, sn_tid_map_by_proj)
 
-                # confirm .tsv file loaded everything as strings
-                types = dict(df.dtypes)
-                unique_types = set([types[k] for k in types])
-                bad_types_present = False
-                for type in unique_types:
-                    if type != object:
-                        bad_types_present = True
-                        break
+        for prep_file in files_list:
+            df = pd.read_csv(prep_file, delimiter='\t')
+            # confirm .tsv file loaded everything as strings
+            types = dict(df.dtypes)
+            unique_types = set([types[k] for k in types])
+            bad_types_present = False
+            for type in unique_types:
+                if type != object:
+                    bad_types_present = True
+                    break
```

Comment:
In read_csv you can add dtype=str and you could avoid all these, no?

Reply:
I think the intention then was to somehow confirm all columns were loaded correctly. I actually think it should be removed because it doesn't actually test anything that can't be forced by dtype=str as you mentioned. It's actually testing pandas more than the code.